frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+
+import {
+  POOL_STORAGE_EFFICIENCY_QUERIES,
+  StorageDashboardQuery,
+} from '../../../../constants/queries';
+import { CompressionItem, SavedItem } from './storage-efficiency-card-item';
+
+import './storage-efficiency-card.scss';
+
+const getGaugeValue = (data) => _.get(data, 'data.result[0].value[1]');
```

Comment:
types

Reply:
Can you move this to utils , so that we can reuse this in other cards ?
We wont miss this way during refactoring this item.

Reply:
ack